evalml/tests/pipeline_tests/regression_pipeline_tests/test_regression.py

```diff
@@ -19,6 +21,21 @@ def test_invalid_targets_regression_pipeline(target_type, dummy_regression_pipel
 
 def test_woodwork_regression_pipeline(linear_regression_pipeline_class):
     X, y = load_diabetes()
-    mock_regression_pipeline = linear_regression_pipeline_class(parameters={'Linear Regressor': {'n_jobs': 1}})
-    mock_regression_pipeline.fit(X, y)
-    assert not pd.isnull(mock_regression_pipeline.predict(X)).any()
+    regression_pipeline = linear_regression_pipeline_class(parameters={'Linear Regressor': {'n_jobs': 1}})
+    regression_pipeline.fit(X, y)
+    assert not pd.isnull(regression_pipeline.predict(X).to_series()).any()
+
+
+def test_custom_indices():
```

Comment:
Moved this from target encoder tests. For what the name / test implies, target encoder isn't necessary for testing, so might as well use OHE (since target encoder won't run when core_dependencies=True)